lib/configure-map.js

```diff
@@ -135,10 +135,24 @@ module.exports = function (original, length, options) {
 			return conf.delete(arg);
 		};
 	}
+	extGet = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return cache[id];
+	};
+	extHas = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return conf.has(id);
+	};
 	defineProperties(memoized, {
 		__memoized__: d(true),
 		delete: d(extDel),
-		clear: d(conf.clear)
+		clear: d(conf.clear),
+		get: d(extGet),
```

Comment:
Let's expose `get` as `_get` (so not as official Public API method). Thing is, that it doesn't work well with `async` mode, as it will return direct result of asynchronous function, and not asynchronously resolved value, as some may expect.
With v1 (hopefully today I'll post new API proposal) it'll be addressed properly